qiskit/primitives/backend_estimator_v2.py

```diff
@@ -87,7 +87,18 @@ class BackendEstimatorV2(BaseEstimatorV2):
     precludes doing any provider- or backend-specific optimizations.
 
     This class does not perform any measurement or gate mitigation, and, presently, is only
-    compatible with Pauli-based observables.
+    compatible with Pauli-based observables. More formally, given an observable of the type
+    :math:`O=\sum_{i=1}^Na_iP_i`, where :math:`a_i` is a complex number and :math:`P_i` is a
+    Pauli operator, the estimator calculates the expectation :math:`E(P_i)` of each :math:`P_i`
+    and finally calculates the expectation value of :math:`O` as
+    :math:`E(O)==\sum_{i=1}^Na_iE(P_i)`. The reported ``std`` is calculated as
```

Comment:
`==` does not look good. Could you use `=`, `:=`, or `\equiv`, etc. as you wish?   
![image](https://github.com/Qiskit/qiskit/assets/31178928/10ff10cd-7357-4d94-b3ad-06238bd7fd84)

Reply:
Oh, thank you for spotting this! It should just be `=`